test/build.info

```diff
@@ -50,7 +50,7 @@ IF[{- !$disabled{tests} -}]
           dtlsv1listentest ct_test threadstest afalgtest d2i_test \
           ssl_test_ctx_test ssl_test x509aux cipherlist_test asynciotest \
           bio_callback_test bio_memleak_test bio_core_test bio_dgram_test param_build_test \
-          bioprinttest sslapitest dtlstest sslcorrupttest \
+          bioprinttest sslapitest sslhandshaketest dtlstest sslcorrupttest \
```

Comment:
This was because of the way that file was linked. We needed to be able to call `ossl_statem_client_write_transition()` directly, which required the linker to be set up differently. Although, I'm interested to know how to skirt around this problem differently.

Reply:
I'm changing this to be called `ssl_handshake_rtt_test`, but if you think of a better name, I'll change it right away

Reply:
Ah, I wrote that comment before actually reading the file closely, so I missed the usage of internal APIs that needed direct access to the static archive.
Thanks for changing the test name!